services/environment-checker.js

```diff
@@ -0,0 +1,19 @@
+function EnvironmentChecker(env, logger, deprecated) {
```

Comment:
The only use you make of this service is to call `logWarnings` to check for inclusion in a list of active environment variables.

Because of this, I feel this could be a singleton service instead of an instantiable one, which directly exports the `logWarnings` method (e.g. how forestadmin/utils/*.js files work). Then, you could have something as direct as `warnOfDeprecatedEnvironment([...]);` in your scripts.